apiserver/clusterservice/scaleimpl.go

```diff
@@ -19,11 +19,13 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"time"
 
 	crv1 "github.com/crunchydata/postgres-operator/apis/cr/v1"
 	"github.com/crunchydata/postgres-operator/apiserver"
 	msgs "github.com/crunchydata/postgres-operator/apiservermsgs"
 	"github.com/crunchydata/postgres-operator/config"
+	"github.com/crunchydata/postgres-operator/events"
```

Comment:
We do'nt need events here. Actions occur synchronously, the user gets immediate feedback.

Reply:
With the move of `startup` and `shutdown` to the `pgo update cluster` command, shutdowns now occur asynchronously.  Therefore, I plan to leave these events, only they will no longer be found in the API server, and will rather be part of the Operator package where the various shutdown logic will now be found.